app/src/fragments/Shell.js

```diff
@@ -1,38 +1,114 @@
 import React, { Component } from 'react'
 import Fab from '@material-ui/core/Fab'
 import AddIcon from '@material-ui/icons/Add'
+import Card from '@material-ui/core/Card'
+import CardActionArea from '@material-ui/core/CardActionArea'
+import CardHeader from '@material-ui/core/CardHeader'
+import Avatar from '@material-ui/core/Avatar'
+import SimpleIcons from 'simple-icons-react-component'
+import { navigate } from '@reach/router'
```

Comment:
Check your linter for warning import and never use

Reply:
I'm sorry I believe this happen while I merge with previous commit, let me fix it